main.go

```diff
@@ -116,3 +129,50 @@ Arguments:
 		helpTab,
 	)
 }
+
+// This section translates a windows path such as "C:\Users\user" to "/Users/user"
+// and removes the default paths for mingw and git4windows to issues when you
+// specify a file path to start code-server in.
+func gitbashWindowsDir(dir string) string {
+	if dir == "~" { //Special case
+		return "~/"
+	}
+	mountPoints := gitbashMountPointsAndHome()
+
+	// Apply mount points
+	absDir, _ := filepath.Abs(dir)
+	absDir = filepath.ToSlash(absDir)
+	for _, mp := range mountPoints {
+		if strings.HasPrefix(absDir, mp[0]) {
+			resolved := strings.Replace(absDir, mp[0], mp[1], 1)
+			flog.Info("Resolved windows path '%s' to '%s", dir, resolved)
+			return resolved
+		}
+	}
+	return dir
+}
+
+// This function returns an array with MINGW64 mount points including relative home dir
+func gitbashMountPointsAndHome() [][]string {
+	// Initialize mount points with home dir
+	mountPoints := [][]string{{filepath.ToSlash(os.Getenv("HOME")), "~"}}
+	// Load mount points
+	out, err := exec.Command("mount").Output()
+	if err != nil {
+		log.Fatal(err)
+	}
+	lines := strings.Split(string(out), "\n")
+	var mountRx = regexp.MustCompile(`^(.*) on (.*) type`)
```

Comment:
Is it safe to assume mount points and paths won't contain spaces? You could do `([^\s]*)` instead to only match non spaces.

Reply:
Unfortunately, no, git4win installs to 
`C:\\Program Files\\Git4Win` or `C:\\Program Files (x86)\\Git4Win` by default enless a user changes it